railties/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Create a `tmp` directory before touching if it does not exist, and keep the tmp directory
+    after cloning.
+    
+    *Yoong Kang Lim*, *Sunny Juneja*
```

Comment:
:heart: very thoughtful of you to credit other contributors here :dancer: 

If you end your commit message(s) with `[Your Name, Another Name]` (on its own line), it'll register the commit for both of you on http://contributors.rubyonrails.org. See http://contributors.rubyonrails.org/faq#credit-several-people